image/Makefile

```
@@ -11,6 +11,7 @@ IMAGE_VERSION                    ?= v0.0.0
 DEBUG                            ?= false
 AUTOLOGIN                        ?= false
 AUTOLOGIN_ARGS                   := $(if $(filter true,$(AUTOLOGIN)),--autologin) # set "--autologin" if AUTOLOGIN is true
+KERNEL_DEBUG_CMDLNE				 := constellation.debug
 export INSTALL_DEBUGD			 ?= $(DEBUG)
```

```suggestion
AUTOLOGIN                        ?= false
AUTOLOGIN_ARGS                   := $(if $(filter true,$(AUTOLOGIN)),--autologin) # set "--autologin" if AUTOLOGIN is true
KERNEL_DEBUG_CMDLNE              := constellation.debug
export INSTALL_DEBUGD            ?= $(DEBUG)
```
Whitespaces are off for these two. Tabs instead of spaces?

---

Yes, this was not intentional.